.kres.yaml

```diff
@@ -25,7 +25,7 @@ spec:
   baseSpecPath: /api
   vtProtobufEnabled: true
   specs:
-    - source: api/storage/storage.proto
+    - source: api/storage/discovery.proto
```

Comment:
Moved to avoid collision in Omni, as it also imports a colliding package/file.proto from Talos.